tests/codebuild/run_canarytest_china.sh

```diff
@@ -48,7 +57,26 @@ function download_installer_china(){
 }
 
 function create_eks_cluster() {
-  eksctl create cluster --name $CLUSTER_NAME --region $CLUSTER_REGION --auto-kubeconfig --timeout=30m --managed --node-type=c5.xlarge --nodes=1
+  if [[ $USE_EXISTING_SUBNET ]]; then
```

Comment:
Here is a more elegant way to optionally add the VPC flags:
https://github.com/kubeflow/pipelines/blob/ada18bc6e604b9eaafa6fdad3e7a671b61606d75/components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests#L105-L107

Reply:
Sweet ! .. thanks for pointing it 
will change

Reply:
done